modules/blocks.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Load code specific to Gutenberg blocks which are not tied to a module.
+ * This file is unusual, and is not an actual `module` as such.
+ * It is included in ./module-extras.php
+ *
+ */
+
+jetpack_register_block(
+	'map',
+	array(
+		'render_callback' => 'jetpack_map_load_assets',
+	)
```

Comment:
Nit: Can we rename `jetpack_map_load_assets` to `jetpack_map_block_load_assets`

Reply:
Done in https://github.com/Automattic/jetpack/pull/10503/commits/ea47243da7b3ed3ec8e0201ff7a237638f807163